doc.go

```diff
@@ -29,13 +29,9 @@ Let's start with an example that shows the sessions API in a nutshell:
 	var store = sessions.NewCookieStore([]byte("something-very-secret"))
 
 	func MyHandler(w http.ResponseWriter, r *http.Request) {
-		// Get a session. We're ignoring the error resulted from decoding an
+		// Get a session. We're ignoring any error that occurs while decoding an
 		// existing session: Get() always returns a session, even if empty.
-		session, err := store.Get(r, "session-name")
-		if err != nil {
-			http.Error(w, err.Error(), 500)
-			return
-		}
+		session, _ := store.Get(r, "session-name")
```

Comment:
This should be the other way around: we should always check the error in example code and remove the 'we're ignoring...' text from the comment.